language/English/strings.po

```diff
@@ -8342,7 +8346,11 @@ msgctxt "#19178"
 msgid "Show channel information when switching channels"
 msgstr ""
 
-#empty string with id 19179
+#. Used as extension header on recordings window
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19179"
+msgid "Deleted recordings trash"
```

Comment:
Was added from me to show that is possible to undelete. If it become removed the also Recordings.

Is currently:
"Recordings - Deleted recordings trash"
if become changed then I use.
"Recordings - Deleted"